Makefile

```diff
@@ -704,6 +708,14 @@ ifneq ($(WIN_REPO_URL), )
 endif
 	$(MAKE) test-conformance CONFORMANCE_E2E_ARGS="-kubetest.config-file=$(KUBETEST_WINDOWS_CONF_PATH) -kubetest.repo-list-path=$(KUBETEST_REPO_LIST_PATH) $(E2E_ARGS)"
 
+## --------------------------------------
+## Security Scanning
+## --------------------------------------
+
+.PHONY: verify-container-images
+verify-container-images: ## Verify container images
```

Comment:
What's the benefit of adding a make target here?

Reply:
One direct benefit is -- we can run `make verify-container-images` locally. Also the actions workflow calls this target which is more cleaner.